.github/workflows/ci-build.yml

```diff
@@ -18,9 +18,6 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
 
-      - name: MongoDB in GitHub Actions
-        uses: supercharge/mongodb-github-action@1.3.0
-
```

Comment:
Not needed anymore. We are using TestContainers in the `QueuedWorkflowRepositoryTest`. That's a Maven dependency that starts a container for that unit test only. We can control when the container is started, destroyed, etc. Later we can think about splitting the tests into groups/suites, so that `mvn test` doesn't need to start the container.